src/main/java/com/synopsys/integration/alert/web/config/PKIXErrorResponseFactory.java

```diff
@@ -47,7 +47,8 @@
     private static final String ALERT_DEPLOYMENT_KUBERNETES = "kubernetes";
 
     private static final String PKIX_HEADER = "There were issues with your Certificates.";
-    private static final String PKIX_TITLE = "To resolve this issue, use the appropriate link below to properly install your certificates and then restart Alert.";
+    private static final String PKIX_TITLE = "To resolve this issue, use the link below to properly install your certificates.";
+    private static final String PKIX_MESSAGE = "Click this link to go to the certificate page: ";
```

Comment:
I feel there is some redundancy here. Could the mssage be shortened to simply be
`Certificate page: `